website/src/App.js

```diff
@@ -37,11 +37,12 @@ export default function App() {
 	return (
 		<ThemeProvider theme={theme}>
 			<div className="app">
-				<div className="nav">
-					<NavBar />
-				</div>
+
 				<div className="content">
 					<BrowserRouter>
+					<div className="nav">
+					<NavBar />
```

Comment:
Can you double check the formatting on this.  Everything should be aligned properly.